miniscope_io/stream_daq.py

```diff
@@ -527,7 +590,8 @@ def capture(
         if video:
             if video_kwargs is None:
                 video_kwargs = {}
-
+            if isinstance(video, str):
+                video = Path(video)
```

Comment:
let's put coercion inside the method and adjust the signature, `str -> Path` is a worthy desire path

Reply:
Committed this, though I'm not sure if I understand the latter part.

Reply:
last part just means "wanting to pass a string to the method will probably be common so it makes sense to coerce to path within that method"